src/requirements.txt

```diff
@@ -1,61 +1,61 @@
 -i https://pypi.python.org/simple
 annotated-types==0.6.0; python_version >= '3.8'
 asgiref==3.7.2; python_version >= '3.7'
-boto3==1.33.7; python_version >= '3.7'
-botocore==1.33.7; python_version >= '3.7'
+boto3==1.34.37; python_version >= '3.8'
+botocore==1.34.37; python_version >= '3.8'
 cachetools==5.3.2; python_version >= '3.7'
-certifi==2023.11.17; python_version >= '3.6'
+certifi==2024.2.2; python_version >= '3.6'
 cfenv==0.5.3
-cffi==1.16.0; python_version >= '3.8'
+cffi==1.16.0; platform_python_implementation != 'PyPy'
```

Comment:
(Q) Just out of curiosity, why was this causing issues? Or was this auto updated?

Reply:
This was auto-updated